src/main.rs

```diff
@@ -13,8 +13,8 @@ pub fn main() {
 
     session.add_event_start(ActivityKind::Notion);
 
-    let notion = cli::Notion::from_args();
-    let exit_code = notion.command.run(&mut session).unwrap_or_else(|err| {
+    let notion: cli::Notion = cli::Notion::from_args();
```

Comment:
I don't think we need the type annotation here.
```suggestion
    let notion = cli::Notion::from_args();
```